src/app/components/article/MediaArticlesDisplay.js

```diff
@@ -29,21 +29,23 @@ const MediaArticlesDisplay = ({ projectMedia, onUpdate }) => {
   return (
     <div className={styles.mediaArticlesDisplay}>
       { hasFactCheck ?
-        <MediaArticleCard
-          key={factCheck.id}
-          variant="fact-check"
-          title={factCheck.title || factCheck.claim_description.description}
-          url={factCheck.url}
-          languageCode={factCheck.language !== 'und' ? factCheck.language : null}
-          date={new Date(factCheck.updated_at * 1000)}
-          statusColor={currentStatus ? currentStatus.style?.color : null}
-          statusLabel={currentStatus ? currentStatus.label : null}
-          publishedAt={publishedAt}
-          id={factCheck.claim_description.id}
-          onClick={() => { setArticleToEdit(factCheck); }}
-          onRemove={onUpdate}
-          removeDisabled={projectMedia.type === 'Blank'}
-        />
+        <>
```

Comment:
question: why wrap a single node in a react fragment?

Reply:
because I forgot to delete it when i was playing around @amoedoamorim ! obviously! 🫠  Thanks for catching this

Reply:
fixed